astropy/visualization/wcsaxes/tests/test_coordinate_helpers.py

```diff
@@ -163,3 +165,15 @@ def test_get_position():
     assert ax.coords[1].get_ticklabel_position() == ["r", "l"]
     assert ax.coords[0].get_axislabel_position() == ["t"]
     assert ax.coords[1].get_axislabel_position() == ["r"]
+
+
+def test_deprecated_getters():
+    fig = plt.figure()
```

Comment:
please use `plt.subplots()` instead. `plt.figure`; the latter isn't thread safe and may create hard-to-debug situations when the test suite is run with multi-threading.